quoting-service/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Quoting-Service Helm chart for Kubernetes
 name: quoting-service
-version: 8.7.1
+version: 8.7.0
 appVersion: "8.7.0-snapshot"
```

Comment:
shouldnt this be version v9.1.0-snapshot? what is what you have in mojaloop values.yaml. You would need to update the values.yaml in the quoting-service folder